common4j/src/main/com/microsoft/identity/common/java/eststelemetry/SchemaConstants.java

```diff
@@ -54,6 +56,17 @@ public static final class Key {
         public static final String FRT_STATUS = TelemetryEventStrings.Key.FRT_STATUS;
         public static final String MRRT_STATUS = TelemetryEventStrings.Key.MRRT_STATUS;
         public static final String ALL_TELEMETRY_DATA_SENT = "is_all_telemetry_data_sent";
+
+        // flw and multiple reg fields
```

Comment:
Should we add comments for each of these fields to tell what they are suppose to capture, to make it easier to read later

Reply:
Added comment with link to doc: https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/1799/commits/870005d2eb30b72568d2298bbc0baa9eb0b98fae